src/erlastic_search.erl

```diff
@@ -411,16 +413,10 @@ bulk_index_docs(IndexTypeIdJsonTuples) ->
     bulk_index_docs(#erls_params{}, IndexTypeIdJsonTuples).
 
 %% Documents is [ {Index, Type, Id, Json}, {Index, Type, Id, HeaderInformation, Json}... ]
+%% calls bulk_operation fun with operation as index
```

Comment:
Shouldn't be a doc comment: users of this lib don't need to know (nor should they care) about internal implementation :)